TESTS.rst

```diff
@@ -1,19 +1,54 @@
 Software tests
 ==============
 
+The Strawberry Fields test suite requires `pytest <https://docs.pytest.org/en/latest/>`_ and `pytest-cov <https://pytest-cov.readthedocs.io/en/latest/>`_ for coverage reports. These can both be installed via ``pip``:
+::
+
+    $ pip install pytest pytest-cov
+
+
 To ensure that Strawberry Fields is working correctly after installation, the test suite can be run by navigating to the source code folder and running
 ::
 
-  make test
+    $ make test
 
-Note that this runs *all* of the tests, using *all* available backends, so can be quite slow (it should take around 40 minutes to complete). Alternatively, you can run the full test suite for a particular backend by running
+Note that this runs *all* of the tests, using *all* available backends, so can be quite slow (it should take around 40 minutes to complete). Alternatively, you can run the full test suite for a particular component by running
```

Comment:
Is the 40 minute estimate accurate for tests that a user would run locally (i.e., in sequence)?

Reply:
Actually, you're right. It would be about an hour now. This is because the previous version of `make test` *said* it was running all the tests, but would only run the tests in pure mode. Now, they will run in both pure and mixed mode.

Reply:
```suggestion
Note that this runs *all* of the tests, using *all* available backends, in both pure *and* mixed modes, so can be quite slow (it should take around 1 hour to complete). Alternatively, you can run the full test suite for a particular component by running
```